uniffi_udl/src/converters/mod.rs

```diff
@@ -54,10 +54,14 @@ impl APIConverter<VariantMetadata> for weedle::interface::OperationInterfaceMemb
         }
         let name: String = {
             use weedle::types::{
-                NonAnyType::Identifier, ReturnType, SingleType::NonAny, Type::Single,
+                NonAnyType::{self, Identifier},
+                ReturnType,
+                SingleType::NonAny,
+                Type::Single,
             };
             match &self.return_type {
                 ReturnType::Type(Single(NonAny(Identifier(id)))) => id.type_.0.to_owned(),
+                ReturnType::Type(Single(NonAny(NonAnyType::Error(_)))) => "Error".to_string(),
```

Comment:
Yep this is exactly what I did when I was trying to build/test a fix for this locally. Great to see the test with it 💯